cumulusci/tasks/salesforce/ListCommunities.py

```diff
@@ -0,0 +1,42 @@
+from rst2ansi import rst2ansi
+from cumulusci.tasks.salesforce import BaseSalesforceApiTask
+
+
+class ListCommunities(BaseSalesforceApiTask):
+    api_version = "46.0"
+    task_docs = """
+    Lists Communities for the current org via the Connect API.
+    """
+    task_options = {}
+
+    def _init_options(self, kwargs):
+        super(ListCommunities, self)._init_options(kwargs)
+
+    def _run_task(self):
+        community_list = self.sf.restful("connect/communities")["communities"]
+        communities = [c for c in community_list]
```

Comment:
Just do:

    communities = self.sf.restful("connect/communities")["communities"]

If that doesn't produce something that can be iterated over by `for community in communities:` I'll be surprised.